src/sa_web/static/js/views/app-view.js

```diff
@@ -717,7 +766,7 @@ var Shareabouts = Shareabouts || {};
 
       // Otherwise, assume we have a model ID.
       modelId = model;
-      model = this.places.get(modelId);
+      model = this.collection.place[datasetId].get(modelId);
```

Comment:
Let's make sure we finish this multi-dataset refactor across the AppView. For example, on line 777 below, we are getting an error because we are still using `this.places`.